story/cli.py

```diff
@@ -5,36 +5,35 @@
 import subprocess
 import sys
 import time
+import typing
 from urllib.parse import urlencode
 from uuid import uuid4
 
 import click
-
 import click_help_colors
-
-import click_spinner
-
 import emoji
-
+from blindspin import spinner
 from raven import Client
-
-import requests
+from requests import Session
 
 from .helpers.didyoumean import DYMGroup
 from .version import version
 
+# Initiate requests session, for connection pooling.
+requests = Session()
+
+# Typing hints.
+Content = typing.Union[str, typing.Mapping, typing.List]
 
 if not os.getenv('TOXENV'):
     enable_reporting = True
-    sentry = Client(
-        'https://007e7d135737487f97f5fe87d5d85b55@sentry.io/1206504'
-    )
+    sentry = Client('https://007e7d135737487f97f5fe87d5d85b55@sentry.io/1206504')
 else:
     enable_reporting = False
     sentry = Client()
 
 data = None
-home = os.path.expanduser('~/.asyncy')
+home = os.path.expanduser('~/.storyscript')
```

Comment:
```suggestion
home = os.path.expanduser('~/.story')
```

`.storyscript` might be used by the actual Storyscript compiler.